enforcer-rules/src/it/bom-project-using-spring-repository/verify.groovy

```diff
@@ -14,6 +14,6 @@ assert !buildLog.text.contains("NullPointerException")
 
 // 4 linkage errors are references to java.util.concurrent.Flow class, which does not exist in
 // Java 8 runtime yet.
-def expectedErrorCount = System.getProperty("java.version").startsWith("1.8.") ? 850 : 846
+def expectedErrorCount = System.getProperty("java.version").startsWith("1.8.") ? 841 : 847
```

Comment:
why did this one go up from 846 to 847?

Reply:
I think That's typo. It should be 837 (-9).